File: ide_copy/pset1/mario_less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    //Accepting integer input between 1 and 8
    //include 1 and 8
    do
    {
        //integr input function from cs50
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8); 
    //generating pattern
    for (int i = 0; i < n; i++)
    {
        //printing space
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        //printing # 
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        //goto new line after # is printed
        printf("\n");
    }
}

File: ide_copy/pset6/mario/more/mario.py
from cs50 import * 

# Input loop
while True:
    # Using cs50.get_int to handle integral
    # and non integral inputs
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# Printing pattern
for i in range(n):
    
    # Printing initial set of spaces
    for j in range(n - i - 1):
        print(" ", end="")
    
    # Printing #
    for j in range(i + 1):
        print("#", end="")
        
    # Printing 2 space seperator
    print("  ", end="")
    
    #printing #
    for j in range(i + 1):
        print("#", end="")
    
    # Printing to new line
    print()